blueocean-config/pom.xml

```diff
@@ -19,9 +19,13 @@
       <groupId>${project.groupId}</groupId>
       <artifactId>blueocean-web</artifactId>
     </dependency>
-      <dependency>
-          <groupId>${project.groupId}</groupId>
-          <artifactId>blueocean-commons</artifactId>
-      </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>blueocean-rest-impl</artifactId>
```

Comment:
@michaelneale Where is the circular dep? I'm quite certain that's not the case.

The only other way of doing this would be to start implementing some extension points or something, which seems to me like over engineering/thinking.

One thing that I was thinking though was that the likes of this module needing to depend on `blueocean-rest-impl` does seem wrong to me. I'd have thought it should depend on `blueocean-rest` and that it should provide some factory methods for accessing the active user etc, using the write `@Extension` to create the `BlueUser` instance etc. As far as I can see, that's not possible with the current structure i.e. you need to use `blueocean-rest-impl` directly. Seems wrong to me.